src/Native/System.Net.Http.Native/CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
 project(System.Net.Http.Native)
 
-find_library(CURL NAMES curl)
-if(CURL STREQUAL CURL-NOTFOUND)
-    message(SEND_ERROR "!!! Cannot find libcurl and System.Net.Http.Native cannot build without it. Try installing libcurl4-openssl-dev (or the appropriate package for your platform) !!!")
-    return()
-endif()
+find_package(CURL)
+if(NOT CURL_FOUND)
+    message(FATAL_ERROR "!!! Cannot find libcurl and System.Net.Http.Native cannot build without it. Try installing libcurl4-openssl-dev (or the appropriate package for your platform) !!!")
+endif(NOT CURL_FOUND)
 
```

Comment:
There are 2 issues under umbrella of CMake usage on NetBSD (with pkgsrc).

Is there need to split it into 2 commits? Or leave it as it is.